integration/client/snapshot_unix_test.go

```diff
@@ -0,0 +1,35 @@
+//go:build !windows
+// +build !windows
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package client
+
+import (
+	"testing"
+
+    . "github.com/containerd/containerd"
```

Comment:
Nit: you don't have to `.` import containerd. Do you?

Reply:
integration/client has its own go.mod, so I suspect you do. Certainly the existing files I checked do this too.

Reply:
> integration/client has its own go.mod, so I suspect you do. Certainly the existing files I checked do this too.

Quite right. Completely forgot about the ```go.mod``` in integration/client.